lib/test-runner.js

```diff
@@ -21,50 +21,27 @@
 
 module.exports = {
   'exe': 'node',
-  'args': [__filename, '<cwd>', '<baseUrl>', '<testPage>', '<id>'],
+  'args': [__filename, '<testPage>', '<baseUrl>', '<id>'],
```

Comment:
I don't actually know what this is / what it needs to agree with. I assume it either goes with a `require` in testem's launchers or electron-forge's `start` below. Do we define the order of these args or is that specified by something else (e.g. testem)?

Reply:
Oh, does this go with the following?

```js
// These are the command-line arguments passed to us by test-runner.js
const [ , , , testPageURL, testemURL, testemId ] = process.argv;
```

Reply:
Yeah, I'll add a comment